### PR TITLE
Add coverage workflow

### PR DESCRIPTION
**Pull request by hkctkuy**

### CODE REVIEW

**Comment by SweetVishnya** [comment]
Can you also add a badge?

File: .github/workflows/coverage.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+name: coverage
+
+on:
+  push:
+    branches: [ master, coverage ]
```
Comment (SweetVishnya) [inline .github/workflows/coverage.yaml:5]: Remove coverage branch

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: .github/workflows/coverage.yaml (hunk 1 above)
Comment (hkctkuy) [reply .github/workflows/coverage.yaml]: Done

File: README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 [![aarch64](https://github.com/ispras/casr/actions/workflows/aarch64.yml/badge.svg?branch=master)](https://github.com/ispras/casr/actions/workflows/aarch64.yml)
 [![riscv64](https://github.com/ispras/casr/actions/workflows/riscv64.yml/badge.svg?branch=master)](https://github.com/ispras/casr/actions/workflows/riscv64.yml)
 [![fuzzing](https://github.com/ispras/casr/actions/workflows/fuzzing.yml/badge.svg?branch=master)](https://github.com/ispras/casr/actions/workflows/fuzzing.yml)
+[![codecov](https://codecov.io/github/ispras/casr/graph/badge.svg?token=D9VY1WRWA7)](https://codecov.io/github/ispras/casr)
```
Comment (SweetVishnya) [inline README.md:9]: You may add it next to docs badge

**Review by SweetVishnya — CHANGES_REQUESTED** [review]
Don't we want to run tests and collect coverage for them at the same time?

File: README.md (hunk 2 above)
Comment (hkctkuy) [reply README.md]: Done

**Review by SweetVishnya — APPROVED** [review]

**Review by anfedotoff — APPROVED** [review]

**Pull request merged**